auction-server/src/opportunity/entities/opportunity_svm.rs

```diff
@@ -271,9 +293,10 @@ impl TryFrom<repository::Opportunity<repository::OpportunityMetadataSvm>> for Op
                     order_address: program.order_address,
                 })
             }
-            repository::OpportunityMetadataSvmProgram::SwapKamino(program) => {
-                OpportunitySvmProgram::SwapKamino(OpportunitySvmProgramSwap {
+            repository::OpportunityMetadataSvmProgram::Swap(program) => {
+                OpportunitySvmProgram::Swap(OpportunitySvmProgramSwap {
                     user_wallet_address:  program.user_wallet_address,
+                    quote_tokens:         program.quote_tokens,
```

Comment:
aren't these quote token values stored in the Opportunity core fields? why the redundancy of including them in the program field?